### ISSUE TITLE
[BUG] playwright 1.15.0 causes page.goto to throw with 'Frame load interrupted' in webkit

### ISSUE THREAD

**Issue by jhiesey** [microsoft/playwright#9065]
**Context:**
- Playwright Version: 1.15.0
- Operating System: Mac and Linux
- Node.js version: 16.9.1
- Browser: Webkit
- Extra: Our app uses service workers; that may or may not be relevant

**Code Snippet**
```javascript
const { webkit } = require('playwright');

(async () => {
  const browser = await webkit.launch();
  const page = await browser.newPage();
  await page.goto('http://localhost:4000');
  // The previous line throws with
  // 'page.goto: Frame load interrupted'
})();
```

With playwright 1.14.1, this example worked fine. With playwright 1.15.0, `page.goto` rejects with 'page.goto: Frame load interrupted' only on webkit. Firefox and chromium work fine, as does the site when I load it in Safari 15.0 by hand.

Our app does use a service worker which has caused other issues with playwright in the past. I don't know if it's relevant in this case.

**Comment by mxschmitt** [comment]
Could you try if its also reproducible for you with `npm i playwright@next`? Debug logs via the `DEBUG=pw:protocol` environment variable and/or a small repro with the web application would be helpful if you are able to track it more down. Thanks!

**Comment by jhiesey** [comment]
I can try to get a minimal reproduction if you need it, but unfortunately it's not a high priority for me at the moment and could take some time.

**Comment by yury-s** [comment]
I can reproduce it locally with 'https://wormhole.app'

**Comment by mxschmitt** [comment]
This is fixed in 1.15.1
